### ISSUE TITLE
Graal CHANGELOG.md out-of-date.

### ISSUE THREAD

**Issue by fniephaus** [oracle/graal#625]
Graal's [CHANGELOG.md](https://github.com/oracle/graal/blob/master/compiler/CHANGELOG.md) was last updated [on Apr 13, 2017](https://github.com/oracle/graal/commit/3da20b5d084a29c8e860df67d26be45940b04710) when the directory was renamed.

**Comment by dougxc** [comment]
Since the Graal compiler does not export a stable API (and won't for the foreseeable future), this changelog file is not as relevant as Truffle's changelog. For the compiler, it's better to directly use git log to see the changes. For example:
```
git log master --merges --pretty='format:%h %ad %>(20) %an %s' --date=format:'%d.%m.%y %H:%M' .
```
To see changes between GraalVM RC4 and RC5:
```
git log master --merges --pretty='format:%h %ad %>(20) %an %s' --date=format:'%d.%m.%y %H:%M' vm-1.0.0-rc4..vm-1.0.0-rc5
```

**Comment by fniephaus** [comment]
Thanks, @dougxc. That makes sense. I wasn't sure why it hasn't been updated, but now that I know I think it makes sense to remove it completely?

**Comment by dougxc** [comment]
I agree and will do so soon.

**Comment by dougxc** [comment]
Fixed by 812c8bccc0f6d70f3eff211d95252040a56feb04.
